src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -132,7 +132,13 @@ function MoneyRequestParticipantsSelector(
     const isCategorizeOrShareAction = [CONST.IOU.ACTION.CATEGORIZE, CONST.IOU.ACTION.SHARE].some((option) => option === action);
     const [tryNewDot] = useOnyx(ONYXKEYS.NVP_TRY_NEW_DOT, {canBeMissing: true});
     const hasBeenAddedToNudgeMigration = !!tryNewDot?.nudgeMigration?.timestamp;
-    const canShowManagerMcTest = useMemo(() => !hasBeenAddedToNudgeMigration && action !== CONST.IOU.ACTION.SUBMIT, [hasBeenAddedToNudgeMigration, action]);
+    const [optimisticTransactions] = useOnyx(ONYXKEYS.COLLECTION.TRANSACTION_DRAFT, {
+        selector: (items) => Object.values(items ?? {}),
+        canBeMissing: true,
+    });
+    // This is necessary to prevent showing the Manager McTest when there are multiple transactions being created
```

Comment:
Add a blank line before the comment to follow our style guidelines

```suggestion

    // This is necessary to prevent showing the Manager McTest when there are multiple transactions being created
```